samples/sample-app.gradle

```diff
@@ -33,6 +33,13 @@ android {
         // For snapshot builds
         maven { url 'https://s01.oss.sonatype.org/content/repositories/snapshots/' }
     }
+    sourceSets {
+        main {
+            res.srcDirs += [
+                    "${rootDir}/samples/resources",
+            ]
+        }
+    }
```

Comment:
@Shahroz16 I added this to include `appium.xml` for all sample apps so this can be reused. In case you think it doesn't help, please let me know so I can remove it and include for Java sample only.